demo/dlb-init.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+ENABLED_DLB_PF_PCIIDS=${ENABLED_DLB_PF_PCIIDS:-270b 2710 2714}
+NUMVFS=${NUMVFS:-4}
+
+bind() {
+    BSF=$1
+    DRIVER=$2
+
+    LOOP=0
+    while [ "$LOOP" -lt  5 ]
+    do
+        if echo -n 2> /dev/null "$BSF" > /sys/bus/pci/drivers/"$DRIVER"/bind; then
+            break
+        else
+           LOOP=$(("$LOOP" + 1))
+           echo -n 2> /dev/null "$BSF" > /sys/bus/pci/devices/"$BSF"/driver/unbind
+            sleep 1
+        fi
+    done
+}
+
+unbind() {
+    BSF=$1
+    OLD_DRIVER=$2
+
+    LOOP=0
+    while [ "$LOOP" -lt  5 ]
+    do
+        if echo -n 2> /dev/null "$BSF" > /sys/bus/pci/drivers/"$OLD_DRIVER"/unbind;
+        then
+            break
+        else
+           LOOP=$(("$LOOP" + 1))
+            sleep 1
+        fi
+    done
+}
+
+#get dlb pf pci ids
+DEVS=$(for pf in $ENABLED_DLB_PF_PCIIDS; do lspci -n | grep -e "$pf" | grep -o -e "^\\S*"; done)
```

Comment:
The script assumes `dlb2` so we could just get `DEVS` from `realpath  /sys/bus/pci/drivers/dlb2/????\:??\:??\.?`. This also means we don't need to install `lspci` in the image.

Reply:
Ah, ok. :)